recommender/recommender.py

```diff
@@ -968,6 +970,23 @@ def studio_view(self, _context=None):  # pylint: disable=unused-argument
         frag.initialize_js('RecommenderXBlock')
         return frag
 
+    def add_xml_to_node(self, node):
+        """
+        Serialize the XBlock to XML for exporting.
+        """
+        node.tag = 'recommender'
+
+        node.set('intro_enabled', str(self.intro_enabled))
+        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
+        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
+        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
```

Comment:
Then they should remain booleans and ints, shouldn't they? That way, the XML serializer can correctly serialize them. For example, booleans should be serialized to `true` or `false`, not `True` or `False`, as I mentioned below.